whisper/README.md

```diff
@@ -17,9 +17,10 @@ Options:
 	-p, --port PORT                Specify which P2P port to use [default: random].
 	-a, --address ADDRESS          Specify which P2P address to use [default: 127.0.0.1].
 	-s, --secret KEYFILE           Specify which file contains the key to generate the enode.
-    -P, --rpc-port PORT            Specify which RPC port to use [default: 8545].
-    -A, --rpc-address ADDRESS      Specify which RPC address to use [default: 127.0.0.1].
-	-l, --log LEVEL                Specify the logging level. Must conform to the same format as RUST_LOG [default: Error].
+	-P, --rpc-port PORT            Specify which RPC port to use [default: 8545].
+	-A, --rpc-address ADDRESS      Specify which RPC address to use [default: 127.0.0.1].
+	-l, --logging LEVEL            Specify the logging level. Must conform to the same format as RUST_LOG [default: 0]. Log level verbosity configuration choice from highest to lowest priority (error 0, warn 1, info 2, debug 3, trace 4).
```

Comment:
I think we should discourage the use of numeric log levels (I didn't even know they were allowed) and instead only present the string-based level names: `error`, `warn`, `info`, `debug` and `trace`.

Reply:
See my response to your review comment https://github.com/paritytech/parity-ethereum/pull/10810#discussion_r299821365 that addresses this too